conf/sample.ini

```diff
@@ -384,6 +384,9 @@
 # The CSRF check will be executed even if the request has no login cookie.
 ;csrf_always_check = false
 
+# Comma-separated list of plugins ids that won't be loaded inside the frontend sandbox
+;frontend_sandbox_disable_for_plugins =
```

Comment:
I think this better fits in the the [plugins] section (and name it `disable_frontend_sandbox_for_plugins`)

Reply:
This is a security feature that's why is inside the security section.

Reply:
yes, I know but since it only affects plugins  (not grafana in general), I think it suits better in the plugins section (like `app_tls_skip_verify_insecure`). But in any case, here in this section is also configured `angular_support_enabled` which only affects plugins too so there is a precedence. Up to you.